data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
Tfs is a server I hope you know that? ;)
I know alot of games allow public codes, vut they have alot of checks for that code insted or review / check them.
A thing we can't do.

I could crash 1.x with the security checks enabled, and you also seem to think 1.1 is still supported?
1.2 and 1.3 are the only supported version as of right now.
But I agree that 0.x is a complete failure compared to 1.x.

But they are the same thing, either you know what you are doing or your a noob.
A person sith experience that dosnt know what a function does would look it up before using it.
While a noob would hope for the best.

As I said TFS should not be noob proof as we would have to modify every function and class pretty much = waste & more cpp overhead.
Let ppl learn by their mistakes, I know I did.